### ISSUE TITLE
Accessibility can be improved

### ISSUE THREAD

**Issue by bmlancien** [empowerplan/epp-app#144]
I know we are short on time to focus on accessibility, but I still need to point out a few things we could/should improve (in that tool or in the next ones):

- Keyboard navigation: Some elements are not reachable using only a keyboard, which means some users might not be able to use the tool correctly
- HTML structure/Aria labels can be improved in some places (important for screen readers)
- There should be alternative text description for map and charts so that the visually impaired using screen readers also can have access to the main information. It is still a question mark on the what and where, though. Generally, it's about how to make data visualization accessible. Big topic....
- Color contrast needs to be checked again. Using Lighthouse on Chrome I saw that one element could be improved. But there are probably things missing as the URL is the same for the whole app, so I don't know if I got results for the whole app or just a view.
